### PR TITLE
Update tpl_account_history_info_default.php

### PR DESCRIPTION
**Pull request by brittainmark**
Bring in line with 1.5.8b

### CODE REVIEW

File: includes/templates/bootstrap/templates/tpl_account_history_info_default.php [hunk 1]
```diff
@@ -177,9 +177,9 @@
                             </div>
                         </div>
 <?php
-} else { // temporary just remove these 4 lines
+} else {
 ?>
-                        <div>WARNING: Missing Shipping Information</div>
+                        <div><?php echo TEXT_MISSING_SHIPPING_INFO; ?></div>
```
Comment (lat9) [inline includes/templates/bootstrap/templates/tpl_account_history_info_default.php:182]: The associated constant wasn't defined until after Zen Cart 1.5.8a and the template supports the zc157 family as well.

That said, the language files in the `languages/english/extra_definitions` sub-directory (both the zc158 and legacy versions) will require this constant to be added.

**Review by lat9 — CHANGES_REQUESTED** [review]
The change is necessary but insufficient to provide the update.

File: includes/templates/bootstrap/templates/tpl_account_history_info_default.php (hunk 1 above)
Comment (brittainmark) [reply includes/templates/bootstrap/templates/tpl_account_history_info_default.php:182]: Will look at and update.

Comment (brittainmark) [reply includes/templates/bootstrap/templates/tpl_account_history_info_default.php:182]: Added missing language file elements.
Put version check in as over entry in languages/english/account_history_info.php

If you want to leave for 3.7.0 that fine.

File: includes/languages/english/extra_definitions/bootstrap/lang.zca_bootstrap_common.php [hunk 2]
```diff
@@ -100,4 +100,11 @@
     'TEXT_HEADER_ARIA_LABEL_NAVBAR' =>'Navigation Bar',
     'TEXT_HEADER_ARIA_LABEL_LOGO' => 'Site Logo',
 ];
+
+// -----
+// included for account_history_info.php change in i.5.8b
+if (zen_get_zcversion() <= '1.5.8a') {
```
Comment (lat9) [inline includes/languages/english/extra_definitions/bootstrap/lang.zca_bootstrap_common.php:106]: Nope, just add that to the array above; the zc158 language processing will use that as a template-override.

**Review by lat9 — CHANGES_REQUESTED** [review]
No need for version checking.

**Comment by brittainmark** [comment]
I've redone using overrides. Sorry being dense today.
I have tested in 1.5.7 and 1.5.8,

**Review by lat9 — APPROVED** [review]

**Comment by lat9** [comment]
Part of the correction for issue #258.

**Pull request merged**